x-pack/docs/en/security/authorization/run-as-privilege.asciidoc

```diff
@@ -14,7 +14,14 @@ in the LDAP realm, you have to run in
 <<ldap-realm-configuration,_user search_ mode>>. For Active Directory, you need
 to <<ref-ad-settings,configure a `bind_dn` and `secure_bind_password`>>.
 
+.Unsupported realms
+****
 The Kerberos, OpenID Connect, PKI, and SAML realms do not support _run as_.
+Additionally, the `run_as` privilege is supported only when authenticating with
+the `native` or JWT realms, or when using API keys. Tokens that are created with
+the <<security-api-get-token,get token API>> are incompatible with the `run_as`
+privilege because OAuth2 tokens do not support this privilege.
+****
```

Comment:
Making the unsupported realms more salient is good, I guess.
I think the list also include the JWT realm, isn't it @justincr-elastic ?

But I don't think that we should lump in the token mention under the "Unsupported realms" section.
This is a very artificial limitation that is going away soon: https://github.com/elastic/elasticsearch/pull/86680

Reply:
So maybe call it out in one of those "Note" sections, I would suggest.